src/read_sad_lut.F90

```diff
@@ -1225,8 +1228,12 @@ subroutine Read_NCDF_SAD_LUT(Ctrl, LUTFilename, SAD_Chan, SAD_LUT)
             ! Uncertainty = a**2 L**2 + b**2 L + c**2
             call ncdf_read_array(fid, "ru"//char(j+96), chan_tmp_real)
             do i = 1, Ctrl%Ind%NSolar
+               ! ru values stored in LUT already squared?!
                SAD_Chan(Ctrl%Ind%YSolar(i))%Solar%ru2(j) = &
-                    chan_tmp_real(solar_indices(i)) ** 2
+                    chan_tmp_real(solar_indices(i)) !** 2
+               ! Ensure SNR is explicitly set to 0 (used to determine if ru2
+               ! should be used in get_measurments)
+               SAD_Chan(Ctrl%Ind%YSolar(i))%Solar%SNR = 0.0
```

Comment:
Isn't this done on [line 1198](https://github.com/ORAC-CC/orac/blob/master/src/read_sad_lut.F90#L1198)?